pkg/reconciler/openshift/tektonpipeline/extension.go

```diff
@@ -135,10 +136,10 @@ func SetDefault(properties *v1alpha1.PipelineProperties) bool {
 		updated = true
 	}
 
-	// Set `disable-affinity-assistant` to true
-	// webhook will set `false` as default value
-	if properties.DisableAffinityAssistant == nil || !*properties.DisableAffinityAssistant {
-		*properties.DisableAffinityAssistant = DefaultDisableAffinityAssistant
+	// Set `disable-affinity-assistant` to true if not set in CR
+	// webhook will not set any value but by default in pipelines configmap it will be false
+	if properties.DisableAffinityAssistant == nil {
```

Comment:
so from Operator we can configure `DisableAffinityAssistant` only for Openshift and not for K8s then in that case can we add comment here https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonpipeline_types.go#L84 so that it will be more clear

not necessarily in this instead we do it in another PR

WDYT?

Reply:
@savitaashture it's support on all platform. Only on Openshift the default is different (`true` instead of `false`)

Reply:
yep. on kubernetes by default the field will not be set in TektonConfig CR, user can define and alter value.
by default it will be false in configmap.

Reply:
@sm43 but on k8s, if the user set then in the `TektonConfig` CR, the will be applied to the managed tekton instance right ?

Reply:
yes.. user can define the field in k8s and change the value. there is no behavior change